test_run.sh

```diff
@@ -525,6 +525,23 @@ revert_change() {
   mv $1/$2.bak $1/$2
 }
 
+test_intellij_aspect() {
+  intellij_git_tag=$1
+  rules_scala_dir=$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )
+
+  if [[ "$runner" == "run_test_ci" ]]; then
```

Comment:
this feels brittle.  can you detect CI another way?  maybe `test -d intellij`?

Reply:
It doesn't rely on any Travis specific env variables, `ci` is [passed to the script](https://github.com/bazelbuild/rules_scala/pull/309/files#diff-354f30a63fb0907d4ad57269548329e3R44) and then `runner` is set based upon that value so I can't see a situation where it would break nor be able to be handled across multiple ci environments.

Reply:
@kevingessner How about:

`  if  [[ "$1" == "ci" ]]; then`

That is one fewer level of indirection, at least.

Reply:
"brittle" in that, if someone changes the name of the `run_test_ci` runner function, this will not error but will start behaving badly -- it'd be better if this was a reference to the function that was safe against changes.

Reply:
oh yeah, missed that. thanks!
I think @greggdonovan's latest is better.

Reply:
@greggdonovan can you change the above to `if [[ "$1" == "ci" ]]; then` like you suggested?